client/src/components/Libraries/LibraryFolder/TopToolbar/import-to-history/import-dataset.js

```diff
@@ -147,27 +147,23 @@ var ImportDatasetModal = Backbone.View.extend({
      * @param  {array} history_item_set array of empty history items
      * @param  {str} history_name     name of the history to import to
      */
-    chainCallImportingIntoHistory: function (history_item_set, history_name) {
+    chainCallImportingIntoHistory: function (history_item_set, history_name, history_id) {
         const Galaxy = getGalaxyInstance();
         var popped_item = history_item_set.pop();
         if (typeof popped_item == "undefined") {
             if (this.options.chain_call_control.failed_number === 0) {
-                Toast.success("Selected datasets imported into history. Click this to start analyzing it.", "", {
-                    onclick: () => {
-                        window.location = getAppRoot();
-                    },
-                });
+                Toast.success(
+                    "Selected datasets imported into history. Click this to start analyzing it.",
+                    "",
+                    `${getAppRoot()}histories/view?id=${history_id}`
```

Comment:
A bit different wording, and making more use of the space available on the toast.

```suggestion
                    "Click here to start analyzing it.",
                    "Selected datasets imported into history",
                    `${getAppRoot()}histories/view?id=${history_id}`
```

Looks good to me otherwise, and works fine with existing toasts. Thanks for fixing this!